### ISSUE TITLE
[Feature Request] Make a non-italic versions?

### ISSUE THREAD

**Issue by dinhquochan** [subtheme-pro/monokai-pro#15]
Dear,

I really like Monokai Pro, but it's seem use many italic fonts. So, I can switch them manually but can you create another versions for it?

Thanks.

**Comment by jontaydev** [comment]
Yea this has bothered me too. I originally copied the color scheme from Material plugin and they made liberal use of italics, which is frustrating especially if your not using a font like Operator Mono which is designed for beautiful monospaced italics. However, a lot of people like them and I don't want to remove this feature for everybody. I also don't want to have tons of themes to choose from supporting every possible preference.

I need to think of an elegant solution that either makes this configurable from a settings panel or maybe allow users to pass in variables to the build script.

For now you can disable the italics manually for each field, but I will think about this.

**Comment by dinhquochan** [comment]
> I need to think of an elegant solution that either makes this configurable from a settings panel or maybe allow users to pass in variables to the build script.

It's awesome. Thank @jontaydev

**Comment by jontaydev** [comment]
I looked into this and couldn't figure out how to make this changeable from a settings panel. For now, I decided to just add separate color schemes for non-italic versions. I'm not super happy with this - but having a non-italic option was important for me and I know its a divisive issue for even the official theme: https://github.com/Monokai/monokai-pro-vscode/issues/27

Variations are selected from the Editor/Color Scheme settings panel.

<img width="409" alt="Screen Shot 2021-04-27 at 5 46 55 PM" src="https://user-images.githubusercontent.com/2243530/116328277-ac257100-a796-11eb-8166-aedaa35f2537.png">

Uploaded the new version to JetBrains and will be available in a few days. I'm also going to keep researching other solutions to this, so would be happy to hear any ideas if anybody comes across this issue.

Thanks for the feedback.
